examples/decoding/plot_linear_model_patterns.py

```diff
@@ -58,28 +64,49 @@
 # get MEG and EEG data
 meg_epochs = epochs.copy().pick_types(meg=True, eeg=False)
 meg_data = meg_epochs.get_data().reshape(len(labels), -1)
-eeg_epochs = epochs.copy().pick_types(meg=False, eeg=True)
-eeg_data = eeg_epochs.get_data().reshape(len(labels), -1)
 
 ###############################################################################
 # Decoding in sensor space using a LogisticRegression classifier
 
 clf = LogisticRegression()
-sc = StandardScaler()
+scaler = StandardScaler()
 
 # create a linear model with LogisticRegression
 model = LinearModel(clf)
 
 # fit the classifier on MEG data
-X = sc.fit_transform(meg_data)
+X = scaler.fit_transform(meg_data)
 model.fit(X, labels)
-# plot patterns and filters
-model.plot_patterns(meg_epochs.info, title='MEG Patterns')
-model.plot_filters(meg_epochs.info, title='MEG Filters')
 
-# fit the classifier on EEG data
-X = sc.fit_transform(eeg_data)
-model.fit(X, labels)
-# plot patterns and filters
-model.plot_patterns(eeg_epochs.info, title='EEG Patterns')
-model.plot_filters(eeg_epochs.info, title='EEG Filters')
+# Extract and plot spatial filters and spatial patterns
+for name, coef in (('patterns', model.patterns_), ('filters', model.filters_)):
+    # We fitted the linear model onto Z-scored data. To make the filters
+    # interpretable, we must reverse this normalization step
+    coef = scaler.inverse_transform([coef])[0]
+
+    # The data was vectorized to fit a single model across all time points and
+    # all channels. We thus reshape it:
+    coef = coef.reshape(len(meg_epochs.ch_names), -1)
+
+    # Plot
+    evoked = EvokedArray(coef, meg_epochs.info, tmin=epochs.tmin)
+    evoked.plot_topomap(title='MEG %s' % name)
+
+###############################################################################
+# Let's do the same on EEG data using a scikit-learn pipeline
+
+X = epochs.pick_types(meg=False, eeg=True)
+y = epochs.events[:, 2]
+
+# Define a unique pipeline to sequentially:
+clf = make_pipeline(
+    Vectorizer(),                       # 1) vectorize across time and channels
+    StandardScaler(),                   # 2) normalize features across trials
+    LinearModel(LogisticRegression()))  # 3) fits a logistic regression
+clf.fit(X, y)
+
+# Extract and plot patterns and filters
```

Comment:
Might be worth explicitly adding a line that just says "the `inverse_transform` parameter will call this method on any estimators contained in the chain, in reverse order."